test/config.sh

```diff
@@ -174,11 +174,13 @@ globalExcludeTests+=(
 	# single-binary images
 	[hello-world_utc]=1
 	[nats_utc]=1
+        [nats-streaming_utc]=1
 	[swarm_utc]=1
 	[traefik_utc]=1
 
 	[hello-world_no-hard-coded-passwords]=1
 	[nats_no-hard-coded-passwords]=1
+        [nats-streaming_no-hard-coded-passwords]=1
```

Comment:
These look good (although the indentation should be tabs, not spaces), and there's one more further down (`nats_override-cmd` needs to be copied for `nats-streaming_override-cmd`). :+1: